Remotion/Core/Reflection/MethodInfoAdapter.cs

```diff
@@ -45,6 +45,15 @@ public static MethodInfoAdapter Create (MethodInfo methodInfo)
       return s_dataStore.GetOrAdd (methodInfo, s_ctorFunc);
     }
 
+    [CanBeNull]
+    public static MethodInfoAdapter CreateOrNull ([CanBeNull] MethodInfo methodInfo)

```

Comment:
Isn't there actually a resharper annotation that connects null-input to null-output? And for NRT we have NotNullIfNotNullAttribute

Reply:
I could not find one. Reference: https://www.jetbrains.com/help/resharper/Reference__Code_Annotation_Attributes.html#top

Reply:
https://www.jetbrains.com/help/resharper/Reference__Code_Annotation_Attributes.html#ContractAnnotationAttribute
```
// A method that returns null if the parameter is null,
// and not null if the parameter is not null
[ContractAnnotation("null => null; notnull => notnull")]
public object Transform(object data)
```